Makefile

```diff
@@ -7,10 +7,12 @@ format:
 build: 
 	mkdir build logs
 
-SANITIZE ?= OFF
+test: build/universal-robots
+	./build/universal-robots-test
+
 build/universal-robots: format build
 	cd build && \
-	cmake -G Ninja -DENABLE_SANITIZER=$(SANITIZE) .. && \
```

Comment:
this doesn't do anything

Reply:
I grabbed this makefile from an existing C++ module and didn't add in ASAN support yet